doc/bucket-replication.md

```diff
@@ -0,0 +1,82 @@
+[NooBaa Operator](../README.md) /
+
+# Bucket Replication
+Bucket replication is a NooBaa feature that allows a user to set a replication policy for all or some objects. The goal of replication policies is simple - to define a target for objects to be copied to.
+
+When applied to a bucketclass, the policy will apply to all future bucket claims that'll utilize the bucketclass.
```

Comment:
I feel like you tried to clarify something here, but in this context it is confusing - maybe move it and expand the explanation?

Reply:
I think it is better (there is a title and the context is clearer).